source/common/ssl/utility.cc

```diff
@@ -8,6 +8,16 @@
 namespace Envoy {
 namespace Ssl {
 
+const ASN1_TIME& epochASN1_Time() {
+  static ASN1_TIME* e = []() -> ASN1_TIME* {
+    ASN1_TIME* epoch = ASN1_TIME_new();
+    time_t epoch_time = 0;
+    ASN1_TIME_set(epoch, epoch_time);
```

Comment:
Should we be error checking these calls?

Reply:
I think it is not required because, we are passing known values here.

Reply:
If that's the case, an `ASSERT` or `RELEASE_ASSERT` would be fine to document this.